pkg/kubelet/apis/podresources/server_v1_test.go

```diff
@@ -171,12 +202,14 @@ func TestListPodResourcesV1(t *testing.T) {
 			mockCPUsProvider.EXPECT().GetAllocatableCPUs().Return([]int64{}).AnyTimes()
 			mockDevicesProvider.EXPECT().GetAllocatableDevices().Return([]*podresourcesapi.ContainerDevices{}).AnyTimes()
 			mockMemoryProvider.EXPECT().GetAllocatableMemory().Return([]*podresourcesapi.ContainerMemory{}).AnyTimes()
+			mockDynamicResourcesProvider.EXPECT().GetDynamicResources(pods[0], &containers[0]).Return([]*podresourcesapi.DynamicResource{}).AnyTimes()
```

Comment:
shouldn't this return `tc.dynamicResources` ? the reason why the test is still passing seems to be you didn't yet update `equalListResponse` below to check the dynamic resources